drivers/SmartThings/philips-hue/src/init.lua

```diff
@@ -712,7 +714,12 @@ local function do_bridge_network_init(driver, bridge_device, bridge_url, api_key
 
       local bridge_api = bridge_device:get_field(Fields.BRIDGE_API)
       cosock.spawn(function()
-        Discovery.scan_bridge_and_update_devices(driver, bridge_device:get_field(Fields.BRIDGE_ID))
+        -- auto-add/auto-delete is only supported with newer lua libs. We do a conditional
+        -- check in the delete routine, but we try to avoid doing it at all here with an
+        -- additional defensive check against the Lua Libs API version.
+        if api_version >= 9 then
```

Comment:
This is an optimistic mitigation that tries to preserve the feature for 0.52 hubs. This condition *will* be false on 0.47 Tizen hubs.

However, if we're concerned that this is dodgy for any reason, I'm fully in favor of just nuking this function call entirely.

Reply:
Just posting this as me trying to follow along & confirm things, nothing actionable here.

I see this version coming from here: https://github.ecodesamsung.com/iot-hub/scripting-engine/blob/c0a00cd57b3e5c4f6c6b4333ba1d20dd38df9ee2/lua_libs/version.lua#L3

If I look back at 0.47.x, here is what we'd have for v2 / v3: https://github.com/PhysicalGraph/hub-os/blob/3.27.11/meta-smartthings/recipes-smartthings/edge-driver-lua-libs/edge-driver-lua-libs_0.47.bb

And we see that archive here: https://smartthings.jfrog.io/ui/native/edge-driver-libs/release-0.47/

Which contains `version.lua`:
```lua
local version = {      
  rpc = rpc_version(),      
  api = 4,      
}      
      
return version 
```

Reply:
@dljsjr I know that CELL & TV are on lower API versions since @cjswedes just bumped this recently: https://github.ecodesamsung.com/iot-hub/scripting-engine/commit/49eb1132bea4bb102201786029a0cb84fa735ed2

But how can I tell for sure which archive artifact those platforms have?